packages/frontend/src/Views/Home/Pagination/components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { Fragment, useState } from "react";
+import {
+  Wrapper,
+  Title,
+  Tag,
+  RecipeTypeWrapper,
+  RecipeType,
+  Icon,
+  TagsWrapper,
+} from "./Accordion.styled";
+import { useTags } from "src/services/hooks/useTags";
+import plus from "src/assets/images/plus.svg";
+import minus from "src/assets/images/minus.svg";
+import { usePaginationContext } from "src/Views/Home/Pagination/hooks/usePaginationContext";
+import { setTagsArr } from "src/Views/Home/Pagination/reducer/actions/pagination.actions";
+
+export const Accordion: React.FC = () => {
+  const [activeGroups, setActiveGroups] = useState<string[]>([]);
+  const [state, dispatch] = usePaginationContext();
+
+  const { data } = useTags();
+
+  const tagsArrData = data?.data[0].innerTags;
+
+  const toggleOpenRecipeType = (e: React.MouseEvent<HTMLDivElement>) => {
+    const tagType = e.currentTarget.innerText;
```

Comment:
nie używaj inner text jak już tak korzystassz to wydaje mi się że lepiej ustawiać coś za pomocą id